packages/dashboard/src/components/DashboardFrame/index.tsx

```diff
@@ -136,6 +136,7 @@ const ShapeContainer = styled('div')(({ theme }) => ({
 export interface PageFrameProps extends React.PropsWithChildren<{}> {
     title: React.ReactNode | string
     primaryAction?: React.ReactNode
+    withoutShapeContainer?: boolean
```

Comment:
Please contact the designer. If the inconsistency here is by design or a mistake.

Reply:
This is by design